index.html

```diff
@@ -852,7 +852,7 @@ <h2><dfn>CaptureController</dfn></h2>
           </p>
           <pre class="idl">
             [Exposed=Window, SecureContext]
-            interface CaptureController {
+            interface CaptureController : EventTarget {
```

Comment:
Given there is no event handler, it is a bit strange to add EventTarget.
We could add a note stating something like:
```
CaptureController is currently not defining any event handler. As such, it is not required to inherit from EventTarget.
This is added for the benefit of other specifications that extend CaptureController with event handler attributes.
Removal of this inheritance may be envisioned depending on the success of those other specifications.
```

Reply:
Thanks. I've added this note, modulo some stylistic changes. Could you please take a look and let me know if this is something you'd support in the next interim? (Same request to @jan-ivar.)